src/ApiCore/Transport/GrpcTransport.php

```diff
@@ -161,7 +156,9 @@ private function getCallOptions(array $options)
             ? $options['transportOptions']['grpcOptions']
             : [];
 
-        $callOptions += ['call_credentials_callback' => $this->authWrapper->getAuthorizationHeaderCallback()];
+        if (isset($options['authWrapper'])) {
+            $callOptions += ['call_credentials_callback' => $options['authWrapper']->getAuthorizationHeaderCallback()];
```

Comment:
Could this be an assignment instead of a union?

```php
$callOptions['call_credentials_callback'] = $options['authWrapper']->getAuthorizationHeaderCallback();
```